packages/app/src/elements/audit.ts

```diff
@@ -0,0 +1,98 @@
+// import { translate as $l } from "@padloc/locale/src/translate";
+import { animateElement } from "../lib/animation";
+import { customElement, state, queryAll } from "lit/decorators.js";
+import { css, html, LitElement } from "lit";
+
+import { shared, mixins } from "../styles";
+import { Routing } from "../mixins/routing";
+
+// TODO: Improve UI
+// TODO: Improve UX
+// TODO: Use translations
+
+@customElement("pl-audit")
+export class Audit extends Routing(LitElement) {
+    @state()
+    private _reusedPasswordCount: number = 0;
+
+    @state()
+    private _weakPasswordCount: number = 0;
+
+    @state()
+    private _compromisedPasswordCount: number = 0;
```

Comment:
I'm thinking for a "v1" of Password Audit page, to show these 3 counts, and allow seeing items filtered by them.